build/cwc/frameworks.js

```diff
@@ -20,6 +20,23 @@
 let closureBuilder = require('closure-builder');
 let glob = closureBuilder.globSupport();
 
+/**
+ * AIY Framework
+ */
+closureBuilder.build({
+  name: 'cwc.framework.AIY',
+  compress: true,
+  srcs: glob([
+    'src/frameworks/internal/aiy/*.js',
+  ]),
+  deps: [
+    'src/frameworks/internal/runner/runner.js',
+    'src/utils/logger.js',
+    'src/utils/stack_queue.js',
```

Comment:
stack_queue is deprecated use 'third_party/coding-with-chrome-libraries/src/utils/stack/stack.js', instead.